src/AfricasTalkingChannel.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace NotificationChannels\AfricasTalking;
+
+use AfricasTalking\SDK\AfricasTalking as AfricasTalkingSDK;
+use Exception;
+use Illuminate\Notifications\Notifiable;
+use Illuminate\Notifications\Notification;
+use NotificationChannels\AfricasTalking\Exceptions\CouldNotSendNotification;
+
+class AfricasTalkingChannel
+{
+    /** @var AfricasTalkingSDK */
+    protected $at;
+
+    /** @param TwitterOAuth $twitter */
+    public function __construct(AfricasTalkingSDK $at)
+    {
+        $this->at = $at;
+    }
+
+    /**
+     * Send the given notification.
+     *
+     * @param mixed $notifiable
+     * @param \Illuminate\Notifications\Notification $notification
+     * @throws CouldNotSendNotification
+     */
+    public function send($notifiable, Notification $notification)
+    {
+        $message = $notification->toAfricasTalking($notifiable);
+        $driver = $notifiable->routeNotificationFor('AfricasTalking');
```

Comment:
lowercase first letter here please

Reply:
It's also a good idea to check the `routeNotificationFor` method exists on the `notifiable` object